### ISSUE TITLE
Homey desktop app not working on OSX

### ISSUE THREAD

**Issue by BLAST3R** [athombv/homey#1699]
**What is your Homey version (Settings → System)?**

1.3.1

**What did you try to achieve?**

Open the Homey Desktop app

*Please include detailed steps how we can reproduce the issue*

Open the app on OSX Sierra

**What was the result?**

A lot of gibberish in the window, refresh in the window does nothing

**What did you expect as result?**

A functional desktop app

**Any other remarks**

**Please attach a Homey System log (Settings -> System -> Send Log)**

Actually, this has nothing to do with homey, but with the desktop app. Please see the screenshot as to what I am seeing. This also happens after updating the app with the latest downloadable version. Please advise.

![screen shot 2017-07-29 at 15 41 51](https://user-images.githubusercontent.com/733419/28745252-c2973af8-7474-11e7-96f3-d7847bda1579.png)

**Comment by WeeJeWel** [comment]
That's odd. Can you verify your system time, and that you're able to visit https://my.athom.com in Google Chrome?

**Comment by BLAST3R** [comment]
Yes, my.athom.com works correctly - it's how I'm controlling Homey right now. Also do not seem to be having any other troubles connecting to websites or other services/apps. The athom-cli tool also works perfectly (which also connects to the Athom cloud, right?).

I will try to verify if it also does this on my MacBook.

**Comment by BLAST3R** [comment]
Okay, I renamed the Homey folder in ~/Library/Application Support, and that seems to have fixed it. I can zip the old contents of the folder and send it if you want to take a look at it?

**Comment by WeeJeWel** [comment]
That would be great!

**Comment by BLAST3R** [comment]
Where would you want to receive it? Don't know if it contains sensitive account tokens (probably does). Can I mail it to you or Athom directly?

**Comment by Moszkowizz** [comment]
Hi Blaster, you can send the .zip to bram@athom.com (that's my e-mail) and I will make sure it gets to Emile!
